recipes/sasl/meta.yaml

```diff
@@ -0,0 +1,72 @@
+{% set name = "sasl" %}
+{% set version = "0.2.1" %}
+{% set md5 = "ca093d9a3d6f20b79b964a5e5add0202" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/8e/2c/45dae93d666aea8492678499e0999269b4e55f1829b1e4de5b8204706ad9/{{ name }}-{{ version }}.tar.gz
```

Comment:
Please use the canonical url found here:

https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L18